runtime/builtin_types_test.go

```diff
@@ -23,6 +23,32 @@ import (
 	"testing"
 )
 
+func TestBuiltinDelAttr(t *testing.T) {
+	f := NewRootFrame()
+	delattr := mustNotRaise(Builtins.GetItemString(NewRootFrame(), "delattr"))
```

Comment:
That is needed for `SetAttr(f, fooForDelAttr, NewStr("bar"), None)`,
Am I missing something?